### PR TITLE
update instructions for using translations in readme

### PR DESCRIPTION
**Pull request by timcowlishaw**
## Changes in this PR:

Little change to the README.md to make it clearer how to work with translations (ensured we're using manage.py instead of django-admin, and added extra notes on removing fuzzy strings)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,74 +258,77 @@ We're using [the built-in django translation module](https://docs.djangoproject.
...
+5. Compile the translations to a binary file (this should also be run inside a `fab sh` shell):
+  ```
+  python ./manage.py compilemessages
+  ```
 
-```
-msgid "namespace.mytranslation"
-msgstr "This is my translation"
-```
+### Changing existing translations:
```
Comment (jacksonj04) [inline README.md:304]:
```suggestion
### Changing existing translations
```

**Review by jacksonj04 — APPROVED** [review]
Teensy heading tweak, then ship it

File: README.md [hunk 2]
```diff
@@ -258,74 +258,77 @@ We're using [the built-in django translation module](https://docs.djangoproject.
...
 
-4) In the generated `.po` file, find the generated msgid string and add the translation below it
+3. Update the locale file by running the following command in a `fab sh` shell:
 
-```
-msgid "namespace.mytranslation"
-msgstr "This is my translation"
-```
+  ```
+  python ./manage.py makemessages --no-obsolete --add-location file -l en_GB
```
Comment (jacksonj04) [inline README.md:278]: These should all be `python manage.py` or `./manage.py`, not both.
```suggestion
  python manage.py makemessages --no-obsolete --add-location file -l en_GB
```

**Review by jacksonj04 — CHANGES_REQUESTED** [review]

**Review by dragon-dxw — APPROVED** [review]

**Comment by timcowlishaw** [comment]
something weird's happened - this seems to already be merged? closing.

**Pull request closed without merging**